pypi_release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Make sure the VERSION in setup.py is updated before running the script
+
+# Exit when any command fails
+set -e
+
+# Build the source distribution
+echo "******** Preparing pypi package..."
+python setup.py sdist
+
+# Build the source distribution without dependencies added for LI internal use.
+echo "******** Preparing pypi package without dependencies..."
+# Temporarily save setup.py for recover
+cp setup.py setup.py.tmp
+# Rename the pypi package name
+sed -i "" "s/name='detext'/name='li-detext'/" setup.py
+# Remove install_requires entries
+sed -i "" "s/install_requires=\[.*\]/install_requires=[]/g" setup.py
```

Comment:
As I remember, previously we reserve numpy and gast in `install_requires`. Here you clear all dependencies. Will this cause any issue?

Reply:
Thanks. It shouldn't. We include numpy 1.16 as an external dependency internally.